README.md

```diff
@@ -6,31 +6,47 @@ Hound is an extremely fast source code search engine. The core is based on this
 [Regular Expression Matching with a Trigram Index](http://swtch.com/~rsc/regexp/regexp4.html). Hound itself is a static
 [React](http://facebook.github.io/react/) frontend that talks to a [Go](http://golang.org/) backend. The backend keeps an up-to-date index for each repository and answers searches through a minimal API. Here it is in action:
 
-![Hound Screen Capture](screen_capture.gif)
+![Hound Screen Capture](imgs/screen_capture.gif)
 
 ## Quick Start Guide
 
 ### Using Go Tools
 
+0. [Install the Go tools](https://golang.org/doc/install) if you don't have it. 
+
++ After installing Go, make sure go's version is 1.4+ by typing `go version` in the command line. 
++ Set up your [`GOPATH`](https://github.com/golang/go/wiki/GOPATH) to a directory where you want to store all your Go packages. Otherwise
+all packages will be installed to `$HOME/go` by default.
+
 1. Use the Go tools to install Hound. The binaries `houndd` (server) and `hound` (cli) will be installed in your $GOPATH.
 
 ```
 go get github.com/hound-search/hound/cmds/...
 ```
 
-2. Create a [config.json](config-example.json) in a directory with your list of repositories.
+2. Create a [simple-config-example.json](config-examples/simple-config-example.json) in your local file directory. For simplicity, we just showed 
+the simplest configuration for local file search only. (For a more hybrid solution, please refer to [config-example.json](config-examples/config-example.json))
+
+In the simple configuration file, we placed a tensorflow repo at `/home/rfan/tensorflow` path. 
 
-3. Run the Hound server with `houndd` and you should see output similar to:
+3. Run the Hound server with `houndd -addr=localhost:6880 -conf <your config.json file>` and you should see output similar to:
 ```
 2015/03/13 09:07:42 Searcher started for statsd
 2015/03/13 09:07:42 Searcher started for Hound
 2015/03/13 09:07:42 All indexes built!
 2015/03/13 09:07:42 running server at http://localhost:6080
 ```
 
+4. Open your browser at `http://localhost:6880`, you should be able to search the tensorflow repo rapidly like the following.
```

Comment:
To clean things up a bit:
> 4. By default, hound hosts a web ui at `http://localhost:6080 `. Open it in your browser, and start searching.

I don't necessarily think we need another screenshot of the UI here since we include that gif at the top of the README.

Reply:
Done. Removed.